### DISCUSSION TITLE
Version for windows 32 bit

### DISCUSSION THREAD

**Discussion by YaDagis** [mockoon/mockoon#653] [Help] [answered]
Hello! Please, tell me which version is correct for windows 32 bit?🥲

**Comment by 255kb** [comment, marked as answer]
Have you tried to use the setup file? https://github.com/mockoon/mockoon/releases/download/v1.18.1/mockoon.setup.1.18.1.exe
It should contain both 32 and 64 bits versions. 
It it's not working, I'll invite you to open an issue so we can investigate the issue or add this specific arch. 
Thank you
